config/ftbquests/quests/chapters/tools.snbt

```diff
@@ -3018,55 +3019,76 @@
 			x: -4.5d
 			y: -15.5d
 			subtitle: "Close enough"
-			description: ["First tier of jetpacks are made from wood, stone and copper. Wood is the basic one, needed for anything better, second is stone and the best is copper!"]
+			description: [
+				"First tier of jetpacks are made from very common materials, what makes them great for early game flying!"

```

Comment:
`First tier of jetpacks are made from very common materials, what makes them great for early game flying!` -> `First tier of jetpacks are made from very common materials, which makes them great for early game flying!`